packit_service/worker/parser.py

```diff
@@ -132,6 +138,50 @@ def parse_pr_event(event) -> Optional[PullRequestEvent]:
             )
         return None
 
+    @staticmethod
+    def parse_pull_request_comment_event(event) -> Optional[PullRequestCommentEvent]:
+        """ look into the provided event and see if it is github issue event """
+        action = nested_get(event, "action")
+        logger.debug(f"action = {action}")
+        is_issue = nested_get(event, "issue")
+        if not is_issue:
+            logger.info("Not a issue event.")
+            return None
+        pr_id = nested_get(event, "issue", "number")
+        is_pr_comment = nested_get(event, "issue", "pull_request")
+        if not is_pr_comment:
+            logger.info("Not a comment on pull request.")
+            return None
+        if action in ["created", "edited"] and pr_id:
+            base_repo_namespace = nested_get(
+                event, "repository", "owner", "login"
+            )
+            base_repo_name = nested_get(event, "repository",  "name")
+            if not (base_repo_name and base_repo_namespace):
+                logger.warning(
+                    "We could not figure out the full name of the repository."
+                )
+                return None
+            github_login = nested_get(event, "comment", "user", "login")
+            if not github_login:
+                logger.warning("We could not figure out GitHub login name from event.")
+                return None
+            target_repo = nested_get(event, "repository", "full_name")
+            logger.info(f"GitHub issue comment {pr_id} event for repo {target_repo}.")
```

Comment:
I see that Github uses `issue` key for Pull Requests, but let's not confuse users more than needed:

`GitHub issue comment {pr_id}` -> `GitHub PR comment {pr_id}` -> `GitHub PR {pr_id} comment`

Reply:
users won't see that, it's a log message, but good point with the reword

Reply:
@jpopelka The message is seen from OpenShift terminal window in `packit-worker-0`.
I guess it would be nice to see the message for the case would like to see in which point `packit-service` is, like workflow.
Like here https://github.com/packit-service/packit-service/blob/master/packit_service/worker/parser.py#L118 https://github.com/packit-service/packit-service/blob/master/packit_service/worker/parser.py#L190